x-pack/filebeat/module/cisco/asa/test/hostnames.log-expected.json

```diff
@@ -89,9 +89,13 @@
             "192.0.2.134",
             "192.0.2.15"
         ],
+        "related.user": [
+            "type"
+        ],
         "service.type": "cisco",
         "source.address": "192.0.2.134",
         "source.ip": "192.0.2.134",
+        "source.user.name": "type",
```

Comment:
@legoguy1000 
Looks like something happened here causing the icmp type to be picked up as the source username.

Reply:
I'll take a look.

Reply:
fix pushed